src/solver/src/grins_solver.C

```diff
@@ -42,6 +42,7 @@ namespace GRINS
 
   Solver::Solver( const GetPot& input )
     : _max_nonlinear_iterations( input("linear-nonlinear-solver/max_nonlinear_iterations", 10 ) ),
+      _continue_after_max_iterations( input("linear-nonlinear-solver/continue_after_max_iterations", false ) ),
```

Comment:
Make this the last one in the initialization list and your warning should go away.